harbor/repository.go

```diff
@@ -0,0 +1,42 @@
+package harbor
+
+import (
+	"fmt"
+)
+
+type Repository struct {
+	ID          int           `json:"id"`
+	Name        string        `json:"name"`
+	ProjectID   int           `json:"project_id"`
+	Description string        `json:"description"`
+	PullCount   int           `json:"pull_count"`
+	StarCount   int           `json:"star_count"`
+	TagsCount   int           `json:"tags_count"`
+	Labels      []interface{} `json:"labels"`
+	// CreationTime time.Time     `json:"creation_time"`
+	// UpdateTime   time.Time     `json:"update_time"`
```

Comment:
are these comments needed?

Reply:
No I just had these lines commented out so I didn't have to import time which seemed better aesthetically before I had to import fmt anyway. I'll uncomment them

Reply:
fixed